.circleci/config.yml

```
@@ -128,8 +129,8 @@ jobs:
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               echo "Uploading wheels..."
-              pip install --user https://github.com/joerick/libcloud/archive/v1.5.0-s3fix.zip wheelhouse-uploader
-              python -m wheelhouse_uploader upload --provider-name S3 --local-folder wheelhouse/ xanadu-wheels
+              pip3 install --user https://github.com/joerick/libcloud/archive/v1.5.0-s3fix.zip wheelhouse-uploader

```

This seems to work installing both `libcloud` and `wheelhouse-uploader` locally using Linux + Python 3.9.
```suggestion
              pip3 install wheelhouse-uploader
```

---

Nice! So maybe the libcloud bug is fixed.

---

Do you want to merge this suggestion in?